### DISCUSSION TITLE
keyboard-chat

### DISCUSSION THREAD

**Discussion by giscus** [listenzz/RNDemo#5] [Announcements]
# keyboard-chat

React Native 开发指南 | 教程

https://todoit.tech/keyboard-chat.html

**Comment by hujindong007** [comment]
大佬，你好， 我想问下，我现在是iOS里面嵌套RN，里面有一个聊天页面，每次点击底部输入框，弹出键盘的时候，会把整个页面都推上去，导航也被顶上去，我看你的是可以停留的，怎么做到的，感谢解答一下
